packages/pigeon/lib/objc_generator.dart

```diff
@@ -92,8 +92,16 @@ const Map<String, String> _propertyTypeForDartTypeMap = <String, String>{
   'Map': 'strong',
 };
 
-String? _objcTypeForDartType(String type) {
-  return _objcTypeForDartTypeMap[type];
+String? _objcTypeForPrimitiveDartType(String type) {
```

Comment:
It seems confusing that you are calling "Foo" the type in the map, but "Foo *" the type here, even though the usage is otherwise identical.

Reply:
I agree.  I'm cleaning that a bit up in later PRs.  I renamed this `_objcTypePtrForPrimitiveDartType` to show we want a pointer type, not the type without the pointer.